assignments/callbacks.js

```diff
@@ -17,39 +18,80 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
     return cb(arr[0]);
   }
 
-  // Function invocation 
+  // Function invocation
   firstItem(items, function(first) {
     console.log(first)
   });
 
 */
 
-
+//1.
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length)
 }
 
+getLength(items, function(length) {
+  console.log(length);
+})
+
+
+
+//2.
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length-1]);
 }
 
+last(items, function(lastOne){
+  console.log(lastOne);
+})
+
+
+//3.
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x,y)
 }
 
+sumNums(4,5, function(num1, num2){
+  console.log(num1+num2);
+})
+
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x,y);
 }
 
+multiplyNums(4,5, function(num1, num2){
+  console.log(num1 * num2);
+})
+
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  return cb(item, list);
 }
 
+contains('Gum', items, function(item, arr) {
+    console.log(arr.includes(item));
+})
+
```

Comment:
Make sure you look through the solution code and study how they wrote their callback functions. These work, but I think it would benefit you to look over those.